### PR TITLE
Add interest/groups screens to onboarding

### PR DESCRIPTION
**Pull request by lucyxubroad**
## Overview

Added the interest/group screen to onboarding

## Changes Made

- Added a new view controller for the interest / group screen after user selects their Pear goals
- Cleaned up the rest of the code and fixed the group selection screen in onboarding

## Test Coverage

- Manual testing

## Next Steps (delete if not applicable)

- Last step is to add the social media screen

## Related PRs or Issues (delete if not applicable)

#59 

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>

  
  
ezgif com-video-to-gif-19

</details>

### CODE REVIEW

File: CoffeeChat/Cells/SimpleOnboardingTableViewCell.swift
Comment (ywsang) [inline CoffeeChat/Cells/SimpleOnboardingTableViewCell.swift:73]: could we potentially condense the `type` and `subtitle` arguments to just the subtitle? looking at it, it seems like `type = .interest` and having a subtitle always coincide with each other, but I don't know all the scenarios where this cell is used
this would make the cell more generic in case pear ever wants to use this cell with a subtitle, but the cell isn't `type = .interest`

File: CoffeeChat/Models/SimpleOnboardingCell.swift
Comment (ywsang) [inline CoffeeChat/Models/SimpleOnboardingCell.swift:13]: not sure if I missed something, but is this struct only used for the `InterestsGroupsViewController`, and not the other VCs that use the `SimpleOnboardingTableViewCell`? if that's the case, I feel like a different struct name that's specific to the interests/groups would be better, or refactoring so that all the VCs that use `SimpleOnboardingTableViewCell` use this model

**Review by ywsang — CHANGES_REQUESTED** [review]
nothing major, but just a couple questions that i wanna make sure i see the answers to before this gets merged!

File: CoffeeChat/Controller/GroupsViewController.swift
Comment (lucyxubroad) [reply CoffeeChat/Controller/GroupsViewController.swift:143]: This is just because we switched up our search organizations design. So before, it was more complex and Phillip had written code to address that but we don't really need any of that code anymore.

File: CoffeeChat/Controller/InterestsGroupsViewController.swift
Comment (lucyxubroad) [reply CoffeeChat/Controller/InterestsGroupsViewController.swift:11]: It is quite similar but also slightly different so I'm a bit hesitant to just make 1 view controller. But I do think in general, across all the onboarding screens, things can be standardized and combined... Do you think this can be done in our revision phase or do you think I should try to address in this PR / soon?

File: CoffeeChat/Controller/GoalsViewController.swift
Comment (pnor) [inline CoffeeChat/Controller/GoalsViewController.swift:25]: If we can combine `goals` and `selectedGoals` into one variable, we can avoid running into issues if one array doesn't match up

**Review by pnor — APPROVED** [review]
lgtm

File: CoffeeChat/Controller/InterestsGroupsViewController.swift
Comment (ywsang) [reply CoffeeChat/Controller/InterestsGroupsViewController.swift:11]: we can hold off and wait to refactor -- don't worry about it in this PR

File: CoffeeChat/SceneDelegate.swift
Comment (ywsang) [inline CoffeeChat/SceneDelegate.swift:36]: assuming this is for testing purposes? let's make sure it gets set back to checking user defaults before merging

**Review by ywsang — APPROVED** [review]
lgtm! just one tiny thing

**Pull request merged**
